gradle.properties

```diff
@@ -191,3 +191,6 @@ embeddedRedisVersion=0.6
 
 # Used by VersionEye to keep track of dependency versions
 versioneye.projectid=5677b4a5107997002d00131b
+
+cassandraDriverVersion=3.2.0
+cassandraUnitVersion=3.1.4.0-SNAPSHOT
```

Comment:
That's the version where the errors we were having in the other PR have been fixed. I'm chasing them to release a new version.